source/codegen/metadata/nidaqmx/enums.py

```diff
@@ -1156,19 +1156,17 @@
     'PowerUpStates': {
         'values': [
             {
-                'documentation': {
```

Comment:
Mentioned in the PR description, I'm not generating these so they look a little different but the output is the same (Except for the extra enum option).